docs/src/main/asciidoc/build-items.adoc

```diff
@@ -0,0 +1,630 @@
+| +++ If generated, all the Mongo clients need to be unremovable +++
+| https://github.com/quarkusio/quarkus/blob/master/extensions/mongodb-client/deployment/src/main/java/io/quarkus/mongodb/deployment/CodecProviderBuildItem.java[`io.quarkus.mongodb.deployment.CodecProviderBuildItem`, window="_blank"]
+| +++ No JavaDoc found +++
+| https://github.com/quarkusio/quarkus/blob/master/extensions/mongodb-client/deployment/src/main/java/io/quarkus/mongodb/deployment/MongoClientBuildItem.java[`io.quarkus.mongodb.deployment.MongoClientBuildItem`, window="_blank"]
+| +++ No JavaDoc found +++
+| https://github.com/quarkusio/quarkus/blob/master/extensions/mongodb-client/deployment/src/main/java/io/quarkus/mongodb/deployment/BsonDiscriminatorBuildItem.java[`io.quarkus.mongodb.deployment.BsonDiscriminatorBuildItem`, window="_blank"]
+| +++ No JavaDoc found +++
+| https://github.com/quarkusio/quarkus/blob/master/extensions/mongodb-client/deployment/src/main/java/io/quarkus/mongodb/deployment/MongoConnectionNameBuildItem.java[`io.quarkus.mongodb.deployment.MongoConnectionNameBuildItem`, window="_blank"]
+| +++ Holds a Mongo connection name +++
+| https://github.com/quarkusio/quarkus/blob/master/extensions/mongodb-client/deployment/src/main/java/io/quarkus/mongodb/deployment/MongoClientNameBuildItem.java[`io.quarkus.mongodb.deployment.MongoClientNameBuildItem`, window="_blank"]
+| +++ Represents the values of the +++
+| https://github.com/quarkusio/quarkus/blob/master/extensions/mongodb-client/deployment/src/main/java/io/quarkus/mongodb/deployment/MongoConnectionPoolListenerBuildItem.java[`io.quarkus.mongodb.deployment.MongoConnectionPoolListenerBuildItem`, window="_blank"]
+| +++ No JavaDoc found +++
+|===
+== Neo4j client
+[%header,cols=2*]
+|===
+|Class Name |Description
+| https://github.com/quarkusio/quarkus/blob/master/extensions/neo4j/deployment/src/main/java/io/quarkus/neo4j/deployment/Neo4jDriverBuildItem.java[`io.quarkus.neo4j.deployment.Neo4jDriverBuildItem`, window="_blank"]
+| +++ Allows access to the Neo4j Driver instance from within other extensions. +++
+|===
+== Netty
+[%header,cols=2*]
+|===
+|Class Name |Description
+| https://github.com/quarkusio/quarkus/blob/master/extensions/netty/deployment/src/main/java/io/quarkus/netty/deployment/EventLoopSupplierBuildItem.java[`io.quarkus.netty.deployment.EventLoopSupplierBuildItem`, window="_blank"]
+| +++ No JavaDoc found +++
+|===
+== Hibernate ORM with Panache
+[%header,cols=2*]
+|===
+|Class Name |Description
+| https://github.com/quarkusio/quarkus/blob/master/extensions/panache/hibernate-orm-panache/deployment/src/main/java/io/quarkus/hibernate/orm/panache/deployment/PanacheEntityClassBuildItem.java[`io.quarkus.hibernate.orm.panache.deployment.PanacheEntityClassBuildItem`, window="_blank"]
+| +++ Represents a regular Panache entity class. +++
+|===
+== panache/mongodb-panache-common
```

Comment:
How come this has such a "weird" name?

Reply:
My script is trying to read that information from the `quarkus-extension.yaml` in the `runtime` module (https://github.com/quarkusio/jbang-catalog/blob/master/quarkusbuilditemdoc.java#L95).

Reply:
OK, then I guess we need to fix the names for some modules, since `panache/mongodb-panache-common` doesn't really make sense

Reply:
That won't work. Some are not extensions but SPI modules and don't have a `quarkus-extension.yaml`.

I think it would be better to read the `<name>` from the pom file and normalize it a bit (remove `Quarkus - `, remove ` - Deployment` but typically keep the `- SPI` part).

Reply:
Right - I initially thought the name was being read from the pom - it would make sense to use that strategy as a fallback I think

Reply:
I don't think it should be a fallback. The POM names are very consistent and it's the best way to have something nice.

Reply:
Done